src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -101,26 +123,70 @@ static bool queryForward(ConnectionInfo* ciP, QueryContextRequest* qcrP, QueryCo
 
 
   //
-  // 2. Render the string of the request we want to forward
+  // 2. Send the request to the Context Provider (and await the reply)
+  //    Render the payload if V1 forward
```

Comment:
Not fully sure (the diff is complex) but I think you are jumping from 2 to 4 in the sequence of comments.

Reply:
Yeah, the old "3" is "Render the payload", which is only done for V1 forwards.
The 4 should be a 3, the 5 a 4 etc. Will decrement the following numbers

Reply:
Fixed in 9db6f1005